vortex-array/src/compress.rs

```diff
@@ -226,6 +215,39 @@ impl<'a> Compressor<'a> {
     }
 }
 
+/// Check that compression did not alter the length of the validity array.
+fn check_validity_unchanged(arr: &Array, compressed: &Array) {
+    let _ = arr;
+    let _ = compressed;
```

Comment:
needed to prevent compiler complaining about unused vars during release compiles